requirements/install.pip

```diff
@@ -48,11 +49,11 @@ pytz==2024.1
     #   feedgenerator
 pyyaml==6.0.1
     # via python-frontmatter
-sentry-sdk[flask]==1.9.0
+sentry-sdk[flask]==2.9.0
     # via -r requirements/install.in
 sgmllib3k==1.0.0
     # via feedparser
-urllib3==1.25.11
+urllib3==1.26.19
```

Comment:
Make sure to apply pip-tools to all requirements files. Should be configured in pre-commit :)